### DISCUSSION TITLE
Scarlet and Violet Pokédex Entries for Previous Gens

### DISCUSSION THREAD

**Discussion by Alpiral** [PokeAPI/pokeapi#1089] [Ideas]
Hello!

I couldn't find Pokédex entries from Scarlet and Violet for gens 1-8. 

If that data is available, I would like to know where I can find it.

If it's not available, do you plan to add it? 

Thanks!

**Comment by Naramsim** [comment]
Most probably if you didn't find it, it's not there. Would you be able to help us fill it in?
